source/frontend/src/features/documents/documentUpload/SelectedDocumentHeader.tsx

```diff
@@ -55,6 +55,14 @@ export const SelectedDocumentHeader: React.FunctionComponent<ISelectedDocumentHe
     [documentTypes],
   );
 
+  const selectedDocumentType = useMemo(() => {
+    const documentType = documentTypes.find(d => Number(document.documentTypeId) === d.id);
```

Comment:
unrelated to these changes, but isn't the documentTypeId a number? at least I think it is on the backend.

Reply:
it is, the problem is that on the form is a string.